plat/common/aarch64/plat_common.c

```diff
@@ -56,11 +56,13 @@ void bl32_plat_enable_mmu(uint32_t flags)
 
 void bl31_plat_runtime_setup(void)
 {
+#if !DEBUG
```

Comment:
With this change, BL31 runtime logs will be interleaved with the OS runtime logs, which may not be what all platforms want. In fact we introduced `bl31_plat_runtime_setup()` to avoid this issue. Therefore this change is unsuitable for a plat/common implementation. You are free to override this in your own platform how you like. If your platform has more than 1 UART, I would recommend using a different UART for TF and OS runtime logs (see `plat/arm/common/arm_bl31_setup.c` for an example of this).